src/bundle/Resources/translations/content_type.en.xliff

```diff
@@ -816,16 +816,6 @@
         <target state="new">Password must contain at least one non-alphanumeric character</target>
         <note>key: field_definition.ezuser.require_at_least_one_non_alphanumeric_character</note>
       </trans-unit>
-      <trans-unit id="283d87c7f3e760a02fde52274ed0a4a8523d2a0d" resname="field_definition.ezuser.require_not_compromised_password">
-        <source>Password must not be contained in a public breach.</source>
-        <target state="new">Password must not be contained in a public breach.</target>
-        <note>key: field_definition.ezuser.require_not_compromised_password</note>
-      </trans-unit>
-      <trans-unit id="99aa2f672dfd9206b58ddf5d78e6a695ceb9ebf1" resname="field_definition.ezuser.require_not_compromised_password_help">
-        <source>This uses the API at %link% to securely check breach data. The password is not transmitted to the API.</source>
-        <target state="new">This uses the API at %link% to securely check breach data. The password is not transmitted to the API.</target>
-        <note>key: field_definition.ezuser.require_not_compromised_password_help</note>
```

Comment:
This translation is from a recent feature and I believe shouldn't be removed. But why extractor is removing it?

Reply:
fixed in https://github.com/ibexa/admin-ui/pull/766/commits/ddaa268323e1d9d11c144d17fe94aae31b6f177c . By default jms translation extractor is not able to extract from `help` option.